### PR TITLE
travis-ci test script

### PR DESCRIPTION
**Pull request by paulvollmer**
#201

### CODE REVIEW

**Comment by paulvollmer** [comment]
now travis will lint all json files. see the log for this commit here https://travis-ci.org/paulvollmer/countries/builds/195813850

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function lintalljson() {
```
Comment (mledoze) [inline test.sh]: Can you please:
1. move this file to a `ci` folder
2. use camel case for functions

**Comment by mledoze** [comment]
@paulvollmer I'm sorry for the delay, I started a review but it appears that you didn't see it, so I reposted my comment.

**Comment by mledoze** [comment]
> now travis will lint all json files

That's a good idea!

File: test.sh (hunk 1 above)
Comment (paulvollmer) [inline test.sh]: @mledoze done...

**Comment by mledoze** [comment]
Annnd it's merged! Thank you very much for your help @paulvollmer

**Comment by paulvollmer** [comment]
if the tests running on the master (current [Build Status](https://travis-ci.org/mledoze/countries) ), we can add the nice travis ci status badge to the readme...

branch `test-travis-ci` =  [Build Status](https://travis-ci.org/mledoze/countries)

**Pull request merged**